iset.mm

```diff
@@ -156672,6 +156678,1280 @@ mathematician Ptolemy (Claudius Ptolemaeus).  This particular version is
+      prmdc seqf simplr nnabscl ffvelrnd zmulcld eqeq1d ifbid breq1d anbi12d
+      cmpt breq2d eleq1d ifeq12d ifeq1d mpteq2dv eqtr4di seqeq3d fveq2d fveq12d
+      ifbieq12d df-lgs ovmpoga mpd3an3 ) AFGZDFGZDHIZAJKLZMIZMHNZDHUBUCZAHUBUCZ
+      OZMUGZMNZDUHUIZPCMUJZUIZPLZNZFGADUKLUVIIUUNUUOOZUUPUUSUVHFUVJUUPOZUURMHFU
+      VKURZUVKULUVKUUQFGZMFGZUURQUUNUVMUUOUUPAUMUNUVLUUQMUOUPUQUVJUUPUSZOZUVDUV
+      GUVPUVBUVCMFUVCFGUVPUTVAUVPURZUVPUUTQZUVAQZUVBQUVJUUOUVOHFGZUVRUUNUUOVBZU
+      VPULZDHVCVDUVJUUNUVOUVTUVSUUNUUOVEUWBAHVCVDUUTUVAVFVGUQUVPRFUVEUVFUVPUAUD
+      PFCMRVLUVQUVPUAVHZRGZOZUWCCUIUWCVIGZUWCJIZJAVJUCZHAVKSLZMVMVNZGZMUVCNZNZA
+      UWCMVOLZJVPLZKLZMVQLZUWCSLZMVOLZNZUWCDWELZKLZMNZFUWEBUWCBVHZVIGZUXDJIZUWM
+      AUXDMVOLZJVPLZKLZMVQLZUXDSLZMVOLZNZUXDDWELZKLZMNZUXCRCFEUXDUWCIZUXEUWFUXO
+      UXBMBUAVIVRUXQUXMUWTUXNUXAKUXQUXFUWGUXLUWSUWMUXDUWCJVSUXQUXKUWRMVOUXQUXJU
+      WQUXDUWCSUXQUXIUWPMVQUXQUXHUWOAKUXQUXGUWNJVPUXDUWCMVOVTTWATUXQWBWCTWDUXDU
+      WCDWEVTWCWFUVPUWDVBZUWEUWFUXBMFUWEUWFOZUWTFGUXAWGGZUXBFGUXSUWGUWMUWSFUXSU
+      WGOZUWHHUWLFUYAULUYAUWKMUVCFUYAURZUYAMUYBWHUUNUWKQZUUOUVOUWDUWFUWGUUNUYCU
+      WIMIZUWIVMIZWIZQZUUNUYDQZUYEQZUYGUUNUWIFGZUVNUYHUUNUWIUUNAVKUUNWBVKRGUUNW
+      JVAWKZWLZUUNURUWIMUOUPUUNUYJVMFGUYIUYLVMWMWNUWIVMUOWOUYDUYEWPVGUUNUWKUYFU
+      UNUWIWGGUWKUYFWRUYKUWIMVMWGWQWSXIWTXAUQUYAJRGZUUNUWHQUYMUYAXBVAUUNUUOUVOU
+      WDUWFUWGXCJAXDUPUQUXSUWGUSZOZUWRMUYOUWRUYOUWQUWCUYOUWPUYOUUNUWOWGGZUWPFGU
+      UNUUOUVOUWDUWFUYNXCUYOJUWCVJUCUSZUYPUYOUYQUWGUXSUYNVBUWFUYQUWGWIUWEUYNUWF
+      UWGUYQUWCXEXFXGXHUWFUYQUYPWRZUWEUYNUWFUWCWGGUYRUWFUWCUWCXJXKUWCXLWSXGXTAU
+      WOXMUPXNUWEUWDUWFUYNUXRUNWKWLUYOURXOUXSUWCFGZJFGUWGQUWDUYSUVPUWFUWCXPXGXQ
+      UWCJUOWOXRUXSUWFUUODHXSZUXTUWEUWFVBUUNUUOUVOUWDUWFYAUVOUYTUVJUWDUWFDHYBZY
+      CUWCDYDYEUWTUXAXMUPUWEUWFUSOURUWDUWFQUVPUWCYJYFXRZYGVUBYHUYSUDVHZFGOUWCVU
+      CPLFGUVPUWCVUCYIYFYKUVPUUOUYTUVERGUUNUUOUVOYLUVOUYTUVJVUAYFDYMUPYNYOUVJUU
+      OUVTUUPQUWAUVJULDHUOUPXRUEUFADFFUFVHZHIZUEVHZJKLZMIZMHNZVUDHUBUCZVUFHUBUC
+      ZOZUVCMNZVUDUHUIZPBRUXEUXFJVUFVJUCZHVUFVKSLZUWJGZMUVCNZNZVUFUXHKLZMVQLZUX
+      DSLZMVOLZNZUXDVUDWELZKLZMNZYTZMUJZUIZPLZNUVIUKFVUFAIZVUDDIZOZVUEUUPVUIVVK
+      UUSUVHVVNVUDDHVVLVVMVBZYPVVNVUHUURMHVVNVUGUUQMVVNVUFAJKVVLVVMVEZTYPYQVVNV
+      UMUVDVVJUVGPVVNVULUVBUVCMVVNVUJUUTVUKUVAVVNVUDDHUBVVOYRVVNVUFAHUBVVPYRYSY
+      QVVNVUNUVEVVIUVFVVNVVHCPMVVNVVHBRUXPYTCVVNBRVVGUXPVVNUXEVVFUXOMVVNVVDUXMV
+      VEUXNKVVNUXFVUSUWMVVCUXLVVNVUOUWHVURUWLHVVNVUFAJVJVVPUUAVVNVUQUWKMUVCVVNV
+      UPUWIUWJVVNVUFAVKSVVPTUUBYQWDVVNVVBUXKMVOVVNVVAUXJUXDSVVNVUTUXIMVQVVNVUFA
+      UXHKVVPTTTTUUCVVNVUDDUXDWEVVOWAWCUUDUUEEUUFUUGVVNVUDDUHVVOUUHUUIWCUUJBUFU
+      EUUKUULUUM $.
+
+    $( Value of the function ` F ` which defines the Legendre symbol at the
+       primes.  (Contributed by Mario Carneiro, 4-Feb-2015.)  (Revised by Jim
+       Kingdon, 4-Nov-2024.) $)
+    lgsfvalg $p |- ( ( A e. ZZ /\ N e. NN /\ M e. NN ) -> ( F ` M ) = if (
```

Comment:
What is the difference compared with the set.mm version, and why is a revision necessary?

Reply:
Added text for both the difference and the reason in mmil.html.